docs/tutorials/upgrading-ev3dev-from-command-line.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrading Ev3dev from the Command Line
+subject: System
+---
+
+{% include icon.html type="warning" %}
+By upgrading ev3dev you are also upgrading the drivers, so there may be changes that break user software. 
```

Comment:
I'm not sure what you expect me to change based on that information. If people get to this page, they have either found it by themselves or been told to come here. In the former case, they probably know what they are doing anyways (and that case will be rare); in the latter, the context from which they came dictates what they are here for, and it's almost certainly to upgrade ev3dev -- that's what the title says, after all.